cumulus_library_opioid/reference_sql/static_builder.sql

```diff
@@ -0,0 +1,62 @@
+-- noqa: disable=all
+-- This sql was autogenerated as a reference example using the library
+-- CLI. Its format is tied to the specific database it was run against,
+-- and it may not be correct for all databases. Use the CLI's build 
+-- option to derive the best SQL for your dataset.
+
+-- ###########################################################
+
+CREATE EXTERNAL TABLE IF NOT EXISTS `cumulus_mhg_dev_db`.`opioid__keywords` (
+    STR STRING
+)
+STORED AS PARQUET
+LOCATION 's3://cumulus-athena-933137588087-us-east-1/results/cumulus_user_uploads/cumulus_mhg_dev_db/opioid/keywords.filtered'
```

Comment:
Do we want URLs like this in the public repo?

Reply:
probably not - i'll  hand wave it for now, but i'll have to change the behavior in the library so this isn't re-introduced